include/maidsafe/common/convert.h

```diff
@@ -0,0 +1,93 @@
+/*  Copyright 2012 MaidSafe.net limited
+
+    This MaidSafe Software is licensed to you under (1) the MaidSafe.net Commercial License,
+    version 1.0 or later, or (2) The General Public License (GPL), version 3, depending on which
+    licence you accepted on initial access to the Software (the "Licences").
+
+    By contributing code to the MaidSafe Software, or to this project generally, you agree to be
+    bound by the terms of the MaidSafe Contributor Agreement, version 1.0, found in the root
+    directory of this project at LICENSE, COPYING and CONTRIBUTOR respectively and also
+    available at: http://www.maidsafe.net/licenses
+
+    Unless required by applicable law or agreed to in writing, the MaidSafe Software distributed
+    under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+    OF ANY KIND, either express or implied.
+
+    See the Licences for the specific language governing permissions and limitations relating to
+    use of the MaidSafe Software.                                                                 */
+
+#ifndef MAIDSAFE_COMMON_CONVERT_H_
+#define MAIDSAFE_COMMON_CONVERT_H_
+
+#include "asio/ip/address.hpp"
+#include "asio/ip/address_v4.hpp"
+#include "asio/ip/address_v6.hpp"
+#include "asio/ip/udp.hpp"
+
+#include "boost/asio/ip/address.hpp"
+#include "boost/asio/ip/address_v4.hpp"
+#include "boost/asio/ip/address_v6.hpp"
+#include "boost/asio/ip/udp.hpp"
+
+namespace maidsafe { namespace common { namespace convert {
```

Comment:
Changed it

On Mon, Feb 9, 2015 at 5:44 PM, Fraser Hutchison <notifications@github.com>
wrote:

> In include/maidsafe/common/convert.h
> <https://github.com/maidsafe/MaidSafe-Common/pull/117#discussion_r24342546>
> :
>
> > +    use of the MaidSafe Software.                                                                 */
> > +
> > +#ifndef MAIDSAFE_COMMON_CONVERT_H_
> > +#define MAIDSAFE_COMMON_CONVERT_H_
> > +
> > +#include "asio/ip/address.hpp"
> > +#include "asio/ip/address_v4.hpp"
> > +#include "asio/ip/address_v6.hpp"
> > +#include "asio/ip/udp.hpp"
> > +
> > +#include "boost/asio/ip/address.hpp"
> > +#include "boost/asio/ip/address_v4.hpp"
> > +#include "boost/asio/ip/address_v6.hpp"
> > +#include "boost/asio/ip/udp.hpp"
> > +
> > +namespace maidsafe { namespace common { namespace convert {
>
> We don't usually combine several namespaces on one line like this; it's
> usually one per line with a blank line in between each. Probably your best
> bet is to use clang-format everywhere.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/maidsafe/MaidSafe-Common/pull/117/files#r24342546>.
>